README.md

```diff
@@ -1 +1,36 @@
-# dataverse-frontend
\ No newline at end of file
+# Dataverse Frontend
+
+[![Tests](https://github.com/IQSS/dataverse-frontend/actions/workflows/test.yml/badge.svg)](https://github.com/IQSS/dataverse-frontend/actions/workflows/test.yml)
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
```

Comment:
Please include `npm install` so absolute novices may follow along?

Reply:
Sure! Thank you!